spec/index.bs

```diff
@@ -2309,7 +2308,7 @@ interface GPUTextureUsage {
                                     : {{GPUTextureDimension/"1d"}}
                                     ::
                                         - |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/width=] must be less than or equal to
-                                            |this|.{{GPUAdapterLimits/maxTextureDimension1D}}.
+                                            |this|.{{GPUSupportedLimits/maxTextureDimension1D}}.
```

Comment:
These should actually be
`|this|.{{GPUDevice/limits}}.{{GPUSupportedLimits/maxTextureDimension1D}}` or
`|this|.{{GPUObjectBase/[[device]]}}.{{device/[[limits]]}}.[=supported limits/maxTextureDimension1D=]`.

Reply:
Done, thanks!